examples/testnet_simple_taco.py

```diff
@@ -38,7 +38,7 @@
     blockchain_endpoint=polygon_endpoint,
     registry=registry,
 )
-ritual_id = 1  # got this from a side channel
+ritual_id = 3  # got this from a side channel
 ritual = coordinator_agent.get_ritual(ritual_id)
 
 # known already authorized signer for ritual 1
```

Comment:
This mentions ritual 1. But more importantly, is this hardcoded Enrico also authorized for ritual 3?

Reply:
✔️ - updated.

> is this hardcoded Enrico also authorized for ritual 3?

It is. I used the `nucypher_dkg` script to authorize the hardcoded Enrico while testing ritual 3 - https://github.com/nucypher/nucypher/blob/development/scripts/hooks/nucypher_dkg.py#L336